docs/safety/ci-strategy.md

```
@@ -88,7 +88,7 @@ libchewing convention), they do not auto-block unless a ratchet exists
 | fuzz soak + corpus | | | ✔ | |
 | Miri | | | ✔ | |
 | overflow release lane | | | ✔ | |
-| Kani | | | ✔ | |
+| Kani | | | dropped | | |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)